domain/src/main/java/by/goods/categories/Bike.java

```diff
@@ -0,0 +1,10 @@
+package by.goods.categories;
+
+import by.goods.Category;
+
+public class Bike extends Category {
+
+    public Bike(String name) {
```

Comment:
one question, could this category (Bike) has a name differrent from "Bike", if not), please think how you can simplify this constructor and category creation in populator class.

Reply:
updated